spec/index.bs

```diff
@@ -736,13 +736,26 @@ For the duration of each scope, the set of [=internal usage=] flags of any given
 A [=subresource=] may transition to new usages at the boundaries between [=usage scope=]s.
 
 This specification defines the following [=usage scopes=]:
-  1. an individual command on a {{GPUCommandEncoder}}, such as {{GPUCommandEncoder/copyBufferToTexture()|GPUCommandEncoder.copyBufferToTexture}}.
-  2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
-  3. the whole {{GPURenderPassEncoder}}.
+
+- Outside of a pass (in {{GPUCommandEncoder}}), each command is one usage scope
+    (e.g. {{GPUCommandEncoder/copyBufferToTexture()}}).
+- In a compute pass, each command is one usage scope.
+    A subresource is "used" in the usage scope if it's accessible by the command.
+    For example, in {{GPUComputePassEncoder/dispatch()}},
+    every subresource in every currently bound bind group is "used" in the usage scope.
+    (Compute pass state-setting commands are trivially-valid usage scopes; see below.)
```

Comment:
I'm confused. I thought we agreed to say that every dispatch makes up a usage scope. Why do we need to say that state-setting is trivially-valid at all?

Reply:
Should we change it so we no longer say:

> On the Queue timeline, there is an ordered sequence of usage scopes. Each item on the timeline is contained within exactly one scope.

Reply:
Yes, if we consider dispatches to be "items" but not `setBindGroup`